emperor/support_files/js/scale-editor.js

```diff
@@ -0,0 +1,96 @@
+define([
+    'jquery',
+    'underscore',
+    'view',
+    'viewcontroller',
+],
+function($, _, DecompositionView, ViewControllers) {
+
+  /**
+   *
+   * @class Scale
+   *
+   * This class represents a range editor defined by the SlickGrid
+   * project.
+   * Note, this object is heavily based on classes in slick.editors.js and in
+   * the documentation that can be found [here]{@link
+   * https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors}.
+   *
+   */
+  function ScaleEditor(args) {
+    /**
+     * Node containing the jQuery slider
+     * @type {Node}
+     */
+    var $input;
+    /**
+     * Node containing the textbox for showing the slider value
+     * @type {Node}
+     */
+    var $viewval;
+    /**
+     * Initial value of the cell being edited.
+     * @type {Float}
+     */
+    var defaultValue;
+    var scope = this;
+
+    /* @constructor */
+    this.init = function() {
+      $viewval = $('<input type="text" value="' + args.item.value +
+                   '" readonly  style="border:0;width:25px;">');
+      var $sliderDiv = $('<div style="width:115px;display:inline-block;' +
+                         'background-color:rgb(238, 238, 238)">');
+      $input = $sliderDiv.slider({
+        range: 'max',
+        min: 0.1,
+        max: 5.0,
+        value: args.item.value,
+        step: 0.1,
+        slide: function(event, ui) {
+          $viewval.val(ui.value);
+          args.item.value = ui.value;
+        }
```

Comment:
Do you think we could subscribe an `onchange`/`changed`/`finshedchanging` callback to this slider so that we don't have to wait to click the grid for the values to change? This can be done similarly to `color-editor`, see this line:
https://github.com/biocore/emperor/blob/new-api/emperor/support_files/js/color-editor.js#L64

The idea behind it, is that once you want to update the value, you just "commit" the changes to the grid, and the visual elements should be updated.